packages/main/src/Tokenizer.ts

```diff
@@ -172,28 +181,23 @@ class Tokenizer extends UI5Element {
 		this._resizeHandler = this._handleResize.bind(this);
 
 		this._itemNav = new ItemNavigation(this, {
-			currentIndex: "-1",
+			currentIndex: -1,
 			getItemsCallback: this._getVisibleTokens.bind(this),
 		});
 
 		this._scrollEnablement = new ScrollEnablement(this);
 	}
 
-	async onBeforeRendering() {
-		if (this.showPopover && !this._getTokens().length) {
-			const popover = await this.getPopover();
-			popover.close();
-		}
-
+	onBeforeRendering() {
```

Comment:
Is such a big change intended?

Reply:
I discussed it with the team and we decided that it is more appropriate in this way.